src/main/java/net/datafaker/providers/base/Image.java

```diff
@@ -0,0 +1,97 @@
+package net.datafaker.providers.base;
+
+import net.datafaker.providers.entertainment.EntertainmentProviders;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.*;
```

Comment:
Started with specific imports but then went wildcard?

Reply:
Hmm, interesting indeed. Let's fix that.